tests/InlineEditBox.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Menu Automated (non-robot) Tests</title>
+
+	<script type="text/javascript" src="boilerplate.js"></script>
+
+	<script type="text/javascript">
+		require([
+			"doh/runner",
+			"dijit/InlineEditBox",
+			"dijit/registry",
+			"dojo/parser",
+			"dojo/domReady!"
+		], function(doh, InlineEditBox, registry, parser){
+
+			doh.register("parse", function parse(){
+				return parser.parse();
+			});
+
+			doh.register("password", [
+				function charsAreHidden() {
+					var password = registry.byId('password'),
+						expected = "password";
+
+					password.set("value", expected);
+
+					doh.isNot(expected, password.displayNode.innerHTML);
```

Comment:
It's  confusing how you named the variable "expected" but it's actually the value that you *don't* expect.   Other than that though this patch looks good to me.